ra-optimisation/rao-commons/src/test/java/com/farao_community/farao/rao_commons/linear_optimisation/iterating_linear_optimizer/IteratingLinearOptimizerTest.java

```diff
@@ -153,9 +153,6 @@ public void optimize() {
         assertTrue(crac.getExtension(ResultVariantManager.class).getVariants().contains(preOptimVariant));
         assertTrue(crac.getExtension(ResultVariantManager.class).getVariants().contains(workingVariants.get(1)));
         assertFalse(crac.getExtension(ResultVariantManager.class).getVariants().contains(workingVariants.get(0)));
-        assertEquals(0, crac.getRangeAction("PRA_PST_BE").getExtension(RangeActionResultExtension.class)
```

Comment:
can you replace this deleted test with  a check that the PST's initial setpoint is set to 0 for both states ? it will allow you to cover some of the code in fillRangeActionResultsWithNetworkValues()

Reply:
done